ynr/apps/api/templates/api/csv_docs.html

```diff
@@ -0,0 +1,26 @@
+{% extends "api/api-base.html" %}
+{% block api_page_title %}
+  <h2 id="csv">CSV / Excel Download</h2>
+{% endblock %}
+{% block api_content %}
+
+  <p>
+    This CSV file contains details of all candidates in the database including contact details
+  parties and elections they have stood in.
+  </p>
+
+  <p>
+  There is a row per candidacy, so if a single person has stood on 3 ballots, they will have 3 rows in the file.
+  </p>
+
+  <p><a href="{{ MEDIA_URL }}candidates-all.csv" class="button">Download all the candidates</a></p>
```

Comment:
We link to the CSV on the `Election` (not `Ballot`) pages, e.g. https://candidates.democracyclub.org.uk/elections/local.warwick.2019-10-29/

The _real_ solution here is something along the lines of the interface I prototyped in #962, but that's for another day

Reply:
This is one of those things where neither of us have any user research to back what we're saying here, but I feel like there is probably a class of "spreadsheet warrior" user who is going to discover the CSV downloads via the front-end, but if the discovery point is via a JSON endpoint, they become essentially invisible.

I agree that the optimal long-term solution here isn't "have a list of >1,000 individual CSVs", but given there's going to be a fairly substantial chunk of time between now and when we arrive at a better solution for that (during which time we're probably going to have more eyes on this than any other time), I think we should provide a stop-gap solution here for finding CSVs via the front-end by reproducing the list we currently generate on https://candidates.democracyclub.org.uk/help/api _somewhere_ .

Reply:
Related: @JoeMitchell  has literally just blogged

> Looking for a spreadsheet of the candidates? That’ll be on [this page](https://candidates.democracyclub.org.uk/help/api)

at https://democracyclub.org.uk/blog/2019/10/29/its-on/

I think that is a good example of a situation where its useful to have a stopgap for a more human-facing way to expose these.

Reply:
True. Although in the case of a local election, that would have to be..

https://candidates.democracyclub.org.uk/elections/local.adur.2016-05-05/
https://candidates.democracyclub.org.uk/elections/local.amber-valley.2016-05-05/
https://candidates.democracyclub.org.uk/elections/local.barnsley.2016-05-05/
...`<snip>`...
https://candidates.democracyclub.org.uk/elections/local.worcester.2016-05-05
https://candidates.democracyclub.org.uk/elections/local.worthing.2016-05-05
https://candidates.democracyclub.org.uk/elections/local.wyre-forest.2016-05-05

..although I accept 1 page with some-hundred links isn't that much nicer than some-hundred pages in that case.

Also remember there are chunks of elections where the page https://candidates.democracyclub.org.uk/elections/parl.2017-06-08/ (for example) _does exist_ , but its not actually linked anywhere because its in the past so its much harder to discover than https://candidates.democracyclub.org.uk/elections/parl.2019-12-12/

Either way, I'm 100% on board with replacing big list o'CSVs with something better but given we don't have something better yet, it seems replacing it with nothing in the meantime is a step back here.

If you're committed to removing it, at least mentioning the two workarounds/discovery mechanisms we've discussed here:

* CSVs are linked from the `/elections` JSON endpoint and
* CSVs are linked from the `/elections` HTML pages

in the documentation for CSVs would be a massive improvement over what is there now.